components/kyma-environment-broker/internal/process/update/conditions.go

```diff
@@ -0,0 +1,24 @@
+package update
+
+import (
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal"
+)
+
+func ForKyma2(op internal.UpdatingOperation) bool {
+	return op.InstanceDetails.ClusterConfigurationVersion == 2
```

Comment:
@piotrmiskiewicz for update operation queue, is it reasonable to take this from the `InstanceDetails`?

Reply:
We need to look for last update kyma or provisioning operation and take RuntimeVersionData from that object. The main difference between upgradeKyma operation and the updating operation is that in the updating operation we need to keep the same version but in the upogradeKyma - we always have to compute the required version (could differ than the last one)